scipy/linalg/_sketches.py

```diff
@@ -88,34 +86,83 @@ def clarkson_woodruff_transform(input_matrix, sketch_size, seed=None):
 
     Notes
     -----
-    This is an implementation of the Clarkson-Woodruff Transform (CountSketch).
-    ``A'`` can be computed in principle in ``O(nnz(A))`` (with ``nnz`` meaning
-    the number of nonzero entries), however we don't take advantage of sparse
-    matrices in this implementation.
+    To make the statement
+
+    .. math:: \|Ax\| \approx \|A'x\|
+
+    precise, observe the following result which is adapted from the
+    proof of Theorem 14 of [2] via Markov's Inequality. If we have
```

Comment:
we need to have `[2]_` to link it to the reference

Reply:
This and your other comment are resolved in the most recent commit. Thanks for letting me know about this!